include/tvm/script/ir_builder/relax/ir.h

```diff
@@ -69,6 +69,23 @@ TVM_DLL void FuncRetStructInfo(const tvm::relax::StructInfo& ret_sinfo);
  */
 TVM_DLL void FuncRetValue(const tvm::relax::Expr& value);
 
+/*!
+ * \brief Add a Relax function or a TIR PrimFunc to the IRModuleFrame.
+ * \param func The function to be added.
+ * \param func_name_hint The name hint of the function to be added.
+ * \note If the function to be added already exists, return its
+ * GlobalVar directly.
+ * \return The global var bound to the added function.
+ */
+TVM_DLL GlobalVar AddFunction(const BaseFunc& func, String func_name_hint);
```

Comment:
I wonder why these two functions are under `relax` folder. Should it be in `ir` folder?

Reply:
ohhh, fixed, thanks for the suggestion!!!